packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -12,6 +12,7 @@ interface ModalProps {
   children?: ReactNode;
   onClose: () => void;
   isOpen: boolean;
+  disablePortal: true;
```

Comment:
So `disablePortal` shouldn't be a prop, it should just always be set on the `Dialog` like so:
![image](https://github.com/beyondessential/tupaia/assets/129009580/507c2372-58df-4993-80d8-37ce394e2268)